securing_apps/topics/client-registration/client-registration-cli.adoc

```diff
@@ -166,7 +166,7 @@ You might want to avoid storing secrets inside a configuration file by using the
 [[_initial_access_and_registration_access_tokens]]
 ==== Initial Access and Registration Access Tokens
 
-Developers who do not have an account configured at the Keycloak server they want to use can use the Client Registration CLI. That is possible when the realm administrator issues a developer an Initial Access Token. It is up to the realm administrator to decide how to issue and distribute these tokens. The realm administrator can limit the maximum age of the Initial Access Token and the total number of clients that can be created with it.
+Developers who do not have an account configured at the {project_name} server they want to use can instead use the Client Registration CLI. This is possible only when the realm administrator issues a developer an Initial Access Token. It is up to the realm administrator to decide how and when to issue and distribute these tokens. The realm administrator can limit the maximum age of the Initial Access Token and the total number of clients that can be created with it.
```

Comment:
The idea here is that you don't need to have an account in order to use Client Registration CLI. You can simple be given an Initial Access Token and authenticate with that.

So 'instead' would not be the tight word here, rather we can say 'also'.

Reply:
I am happy to make that change. I have more comments coming in from QE, so I plan to push another commit tomorrow morning which will contain this change and those.